Lib/test/test_buffer.py

```diff
@@ -4412,6 +4417,12 @@ def test_issue_7385(self):
         x = ndarray([1,2,3], shape=[3], flags=ND_GETBUF_FAIL)
         self.assertRaises(BufferError, memoryview, x)
 
+    @support.cpython_only
+    def test_pybuffer_size_from_format(self):
+        # basic tests
+        for format in ("i", "3s", "0i", " "):
+            self.assertEqual(_testcapi.pybuffer_size_from_format(format),
+                struct.calcsize(format.encode()))
```

Comment:
.encode() is useless and a little bit inconsistenct since format isn't encoded to call pybuffer_size_from_format().

Would you mind to remove the .encode() call (just pass format) but also indent the code so "struct.calcsize..." is aligned to "_testcapi.pybuffer..."?

Reply:
I have also seen that not encoding weirdly works and tests pass locally but will fail on the CI. So I tried to revert the .encode() on struct.calsize().